examples/tr_decrement/DecrementDemo.md

```diff
@@ -2,6 +2,9 @@
 
 This document represents a full demo of creating a TR (single stock total return) index, and then a Decrement index overlay onto it.
 
+NOTE: if you are only trying to create a decrement index on the universe of MerQube supplied TRs, jump to ##Creating the decrement.
+This section only applies if you are also subscribed to create TR indices.
```

Comment:
i didnt want to delete this full document because its still very useful for those with permissions to do this. and, its how I created all the TRs.

Reply:
@mrondon is working on permissioning for each type of index through the UI.